tests/phpunit/CRM/Contribute/Form/SearchTest.php

```diff
@@ -139,18 +139,18 @@ public function testBatchFilter() {
         'expected_qill' => 'Batch Name Is Null',
       ),
       // Case 3: Search for ONLY those contributions which are created from batch ID - $batchID
-      array(
+      [
         'form_value' => array('contribution_batch_id' => $batchID),
```

Comment:
so this is one of it's complaints - seems OK really

Reply:
It is ok.

The buggy linting is the reason my wholesale array formatting update project got stalled. I tried running a script on the whole codebase to switch the array syntax but it got hundreds of linting errors.

Reply:
:-(

Reply:
We're going to need to fix that linter sooner rather than later because more people are going to want to use short array syntax. I think @twomice was the one who first introduced it.

Reply:
let's blame @twomice

Reply:
I was more hoping @twomice would be able to look at that library we're using and see if some upstream updates are available to fix the array issue.

Reply:
nah let's go with blame :-)

Reply:
Boy, that was a long time ago (2012 Woolman sprint, I believe). From memory, we may be using something like https://atom.io/packages/linter-drupalcs for the code-standards linting. Could someone point me to the relevant tree section of the relevant github repo for the code that's currently doing this overzealous linting?

Reply:
@twomice I have a long memory :)

You can do the same linting locally as Jenkins does:

1. Go to a civicrm install made with buildkit. `cd /path/to/civicrm/`
2. Type `civilint` to scan all modified files (e.g. before submitting a PR)
3. Or to lint a specific file type e.g. `civilint CRM/Contact/BAO/Contact.php`

That command calls this script: https://github.com/civicrm/civicrm-buildkit/blob/master/bin/civilint
Which in turn calls the Drupal coder-sniffer: https://github.com/civicrm/civicrm-buildkit/blob/master/bin/civilint#L16

Reply:
@colemanw I think we leapt to conclusions here - I went through & changed ALL instances to the new format & it got through